grpc-utils/src/main/java/com/glegoux/grpc/GrpcServerSimple.java

```diff
@@ -19,16 +21,29 @@ public class GrpcServerSimple implements GrpcServer {
     private Server server;
     private HTTPServer monitoringServer;
 
+    private final List<BindableService> services;
+
+
+    public GrpcServerSimple(List<BindableService> services) {
+        this.services = services;
+    }
+
+    @Override
     public void start(int port, int monitoringPort) throws IOException {
 
         MonitoringServerInterceptor monitoringInterceptor = MonitoringServerInterceptor.create(Configuration.allMetrics());
 
-        server = ServerBuilder.forPort(port)
-            .intercept(monitoringInterceptor)
-            .addService(new HealthStatusManager().getHealthService())
-            .addService(ProtoReflectionService.newInstance())
-            .build()
-            .start();
+        ServerBuilder serverBuilder = ServerBuilder.forPort(port)
+                .intercept(monitoringInterceptor)
+                .addService(new HealthStatusManager().getHealthService())
+                .addService(ProtoReflectionService.newInstance());
+
+        for (BindableService service : services) {
```

Comment:
with lambda functions? Something like:
`services.forEach(service -> serverBuilder.addService(service))`

Reply:
Yes, I can use `services.forEach(serverBuilder::addService);` for more modern Java 8+ syntax, but a good old `loop` works well also 😄.

Reply:
ok